### PR TITLE
fix: Fix the bug of losing events in the case of multiple partitions while error happening

### PR DESCRIPTION
**Pull request by JevanWu**
This pull request should resolve https://github.com/micronaut-projects/micronaut-kafka/issues/629 and it has been tested based on the example attached

### CODE REVIEW

File: kafka/src/main/java/io/micronaut/configuration/kafka/processor/KafkaConsumerProcessor.java [hunk 1]
```diff
@@ -97,19 +97,7 @@
...
-import java.util.HashSet;
-import java.util.Iterator;
-import java.util.List;
-import java.util.Locale;
-import java.util.Map;
-import java.util.Optional;
-import java.util.Properties;
-import java.util.Set;
-import java.util.UUID;
+import java.util.*;
```
Comment (graemerocher) [inline kafka/src/main/java/io/micronaut/configuration/kafka/processor/KafkaConsumerProcessor.java:100]: Could you revert this an not use star imports?

**Review by graemerocher — CHANGES_REQUESTED** [review]
Thanks, would it be possible to write a test?

**Comment by graemerocher** [comment]
Also if you could sign the CLA. Thanks.

File: kafka/src/main/java/io/micronaut/configuration/kafka/processor/KafkaConsumerProcessor.java (hunk 1 above)
Comment (JevanWu) [reply kafka/src/main/java/io/micronaut/configuration/kafka/processor/KafkaConsumerProcessor.java:100]: Sure

Comment (JevanWu) [reply kafka/src/main/java/io/micronaut/configuration/kafka/processor/KafkaConsumerProcessor.java:100]: But it's better to add this rule to the style checking in the future :)

**Comment by larsduelfer** [comment]
Any update when this will be integrated?

**Comment by graemerocher** [comment]
@JevanWu can you resolve the conflicts?

**Comment by larsduelfer** [comment]
@JevanWu would be much appreciated since I also need this bug fix.

**Comment by JevanWu** [comment]
> @JevanWu can you resolve the conflicts?

Done

**Comment by JevanWu** [comment]
Any other updates?

**Comment by graemerocher** [comment]
Thanks for the contribution!

**Pull request merged**